src/Apps/Order/Routes/Shipping2/Hooks/useHandleSaveFulfillmentDetails.ts

```diff
@@ -0,0 +1,129 @@
+import { useOrderTracking } from "Apps/Order/Hooks/useOrderTracking"
+import { useShippingContext } from "Apps/Order/Routes/Shipping2/Hooks/useShippingContext"
+import { useSaveFulfillmentDetails } from "Apps/Order/Routes/Shipping2/Mutations/useSaveFulfillmentDetails"
+import {
+  FulfillmentType,
+  FulfillmentValues,
+} from "Apps/Order/Routes/Shipping2/Utils/shippingUtils"
+import createLogger from "Utils/logger"
+import { CommerceSetShippingInput } from "__generated__/useSaveFulfillmentDetailsMutation.graphql"
+
+const logger = createLogger(
+  "Order/Routes/Shipping2/Hooks/useHandleSaveFulfillmentDetails"
+)
+
+type ExchangeOrGravityError =
+  | Error
+  | {
+      code: string
+      data: string | null | undefined
+      type: string
+    }
+
+export const useHandleSaveFulfillmentDetails = () => {
```

Comment:
Maybe just `useSaveFulfillmentDetails`, since the `handleSaveFulfillmentDetails` return value is just below? No need to double up

Reply:
yeah, why did I do this ... 🤔 ...
- `useSaveFulfillmentDetails` is the `useMutation` hook for just that operation
- This hook includes prepping the values for the different mutation inputs (pickup, ship, artsy shipping) as well as the error handling
I'll try to think about better naming.